src/components/ThreadsList/ThreadInfo.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {Avatar, Box, createStyles, Grid, makeStyles, Theme, Typography} from "@material-ui/core";
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import CreateIcon from '@material-ui/icons/Create';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
```

Comment:
Chcemy używać tej składni? Do projektu są dodane `styled-components`. Jeśli decydujemy się na używanie `makeStyles` to lepiej usunąć SC z dependencies.

Reply:
removed styled-components dependency